build_js.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+git submodule update --init --recursive
+
+DATA_DIR="$(pwd)/data/"
+
+mkdir -p build_js
+cd build_js
+
+cmake ../src \
+    -DBUILD_GUI_VIEWERS=ON \
+    -DBUILD_PYTHON_BINDINGS=OFF \
+    -DBUILD_DATATOOL=OFF \
+    -DBUILD_PTEX_SUPPORT=OFF \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_PREFIX_PATH="$EMSCRIPTEN" \
```

Comment:
The `$EMSCRIPTEN` variable should be explicitly set to the prefix where emscripten was built otherwise it picks up the wrong compiler.

Reply:
Thanks @abhiskk for testing out this PR and support in my absence.  Regarding, the `$EMSCRIPTEN` variable: it is defined by sourcing the installed emscripten distribution's `emsdk_env.sh` file.  so it is in general a good idea to leave that not explicitly set in the script here.  Perhaps it would be good to include in the script a `source /path/to/emsdk_env.sh` statement though and ask people to modify as necessary.